spring-integration-core/src/main/java/org/springframework/integration/support/json/JsonObjectMapperProvider.java

```diff
@@ -61,6 +64,17 @@ else if (boonPresent) {
 	}
 
 	/**
+	 * Return an instance of {@link Jackson2JsonObjectMapper} with the {@link Jackson2JsonObjectMapper#objectMapper}
+	 * field set as the mapper passed by the caller.
+	 * @param mapper the customized mapper that the caller wants to use
+	 * @return Jackson2JsonObjectMapper object that wraps within the customized mapper provided
+	 * @since 5.0.0
+	 */
+	public static JsonObjectMapper<?, ?> getCustomizedJacksonMapper(ObjectMapper mapper) {
+		return new Jackson2JsonObjectMapper(mapper);
```

Comment:
I don't see benefits of this factory method.
You always can call ` new Jackson2JsonObjectMapper(mapper)` manually with the same result.
Also this factory class is JSON-engine agnostic, so it can't advertise factories which depend on the particular library.

I mean let's remove it altogether!